src/React/doc/index.rst

```diff
@@ -88,11 +91,16 @@ For example:
         return <div>Hello {props.fullName}</div>;
     }
 
-
 .. code-block:: html+twig
 
     {# templates/home.html.twig #}
-    <div {{ react_component('MyComponent', { 'fullName': app.user.fullName }) }}></div>
+    {% extends 'base.html.twig' %}
+
+    {% block body %}
+        <div {{ react_component('MyComponent', { 'fullName': number }) }}>
+            Loading... <i class="fas fa-cog fa-spin fa-3x"></i>
+        </div>
```

Comment:
```suggestion
        </div>
        
        {# component living in a subdirectory: "assets/react/controllers/Admin/OtherComponent" #}
        <div {{ react_component('Admin/OtherComponent') }}></div>
```

The name `Admin/OtherComponent` is correct here, right? 😅

Reply:
That's correct ! I just verified it :)